core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -543,6 +546,9 @@
   TABLE_SAMPLER_OPTS("table.sampler.opt.", null, PropertyType.PREFIX,
       "The property is used to set options for a sampler.  If a sample had two options like hasher and modulous, then the two properties "
           + "table.sampler.opt.hasher=${hash algorithm} and table.sampler.opt.modulous=${mod} would be set."),
+  TABLE_SUSPEND_DURATION("table.suspend.duration", "0s", PropertyType.TIMEDURATION,
+      "For tablets belonging to this table: When a tablet server dies, allow the tablet server this duration to revive before reassigning its tablets"
+          + "to other tablet servers."),
```

Comment:
Ok, does it always take the path  ASSIGNED_TO_DEAD->SUSPENDED->ASSIGNED, even when the setting is 0?